### PR TITLE
Implement a dialog to modify the flex item atrributes when an view holder is  clicked

### PR DESCRIPTION
**Pull request by thagikura**

### CODE REVIEW

**Comment by thagikura** [comment]
@yaraki could you have a look?

File: app/src/main/java/com/google/android/flexbox/recyclerview/FlexItemAdapter.java [hunk 1]
```diff
@@ -51,10 +56,10 @@ public FlexItemViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
 
     @Override
     public void onBindViewHolder(FlexItemViewHolder holder, int position) {
-        holder.mTextView.setText(String.valueOf(position + 1));
-        holder.mTextView.setBackgroundResource(R.drawable.flex_item_background);
-        holder.mTextView.setGravity(Gravity.CENTER);
-        holder.mTextView.setLayoutParams(mLayoutParams.get(position));
+        int adapterPositino = holder.getAdapterPosition();
```
Comment (yaraki) [inline app/src/main/java/com/google/android/flexbox/recyclerview/FlexItemAdapter.java]: typo s/adapterPositino/adapterPosition/

Comment (thagikura) [inline app/src/main/java/com/google/android/flexbox/recyclerview/FlexItemAdapter.java]: Done

**Comment by thagikura** [comment]
Thanks! Fixed.

**Pull request merged**